lnbits/core/views/api.py

```diff
@@ -421,34 +420,18 @@ async def subscribe_wallet_invoices(request: Request, wallet: Wallet):
     logger.debug(f"adding sse listener for wallet: {uid}")
     api_invoice_listeners[uid] = payment_queue
 
-    send_queue: asyncio.Queue[Tuple[str, Payment]] = asyncio.Queue(0)
-
-    async def payment_received() -> None:
-        while True:
-            try:
-                async with async_timeout.timeout(1):
-                    payment: Payment = await payment_queue.get()
-                    if payment.wallet_id == this_wallet_id:
-                        logger.debug("sse listener: payment received", payment)
-                        await send_queue.put(("payment-received", payment))
-            except asyncio.TimeoutError:
-                pass
-
-    task = asyncio.create_task(payment_received())
```

Comment:
haha, what a useless task this was! good catch

Reply:
why useless?`

Reply:
what use did it have? It seems like it just added an extra queue values had to unnecessarily pass through

Reply:
> what use did it have?
- I do not know
- but also I couldn't figure out where messages are added to `payment_queue` (or to `api_invoice_listeners`). I could only locate `get/pop/items` calls.